Backpack-SwiftUI/Spinner/Classes/BPKSpinner.swift

```diff
@@ -69,6 +69,13 @@ public struct BPKSpinner: View {
     }
 }
 
+internal extension BPKSpinner {
```

Comment:
super ultra minor.
As this is only used for the button, could it be a private extension there instead? or is it something that may be used by other components as well? (but always remain internal to bpk)

Reply:
It's currently only for buttons. might as well move it there :D!